web/content/release-3.7.adoc

```diff
@@ -20,4 +20,4 @@ Use the top menu bar for navigation for all guides.
 === Upgrading
 
 * link:/{FOREMAN_VER}/Release_Notes/index-katello.html[Release notes for Katello on RHEL/CentOS]
-* link:/{FOREMAN_VER}/Upgrading_and_Updating/index-katello.html[Upgrading and Updating Katello on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Upgrading_Project/index-katello.html[Upgrading Katello on RHEL/CentOS]
```

Comment:
I had expected this to also match the navigation and mention an upgrade is to 3.7. Then also add a link to the `Updating_Project` guide. Perhaps also update the header.